instrument-starters/opentracing-spring-cloud-redis-starter/src/main/java/io/opentracing/contrib/spring/cloud/redis/RedisTracingProperties.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright 2017-2020 The OpenTracing Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.opentracing.contrib.spring.cloud.redis;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ *
+ * @author Luram Archanjo
+ */
+@ConfigurationProperties("opentracing.spring.cloud.redis")
+public class RedisTracingProperties {
+
+  private boolean enabled = true;
+
+  private String prefixOperationName;
```

Comment:
The config property should be documented in `additional-spring-configuration-metadata.json`

Reply:
It is done, sorry for the delay!